pkgs/top-level/all-packages.nix

```diff
@@ -3120,6 +3120,8 @@ in
   birdfont = callPackage ../tools/misc/birdfont { };
   xmlbird = callPackage ../tools/misc/birdfont/xmlbird.nix { stdenv = gccStdenv; };
 
+  bitflowers = libsForQt5.callPackage ../applications/blockchains/bitflowers/default.nix { };
```

Comment:
Are you using qt4 or qt5? You are mixing right now.

Reply:
The install instructions have both, so to be honest I'm not really sure.

Reply:
The install instructions may be out of date. I would suggest try packaging with qt5 only and see how far you can go.